_commands/basics/time.md

```diff
@@ -0,0 +1,39 @@
+---
+---
+
+time
+--
+
+`time` writes a message to standard output giving timing statistics about the program that was run.
+
+~~~ bash 
+$ time ./executable
+real    0m0.116s
+user    0m0.004s
+sys     0m0.008s
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+`time [-p] command [arguments...]`
+
+`time` can be used with the flag `-p` to print the timing summary in a portable POSIX format
+
+~~~bash
+$ time -p ls
+real    0m0.116s
+user    0m0.004s
+sys     0m0.008s
+~~~
+
+### Output Description
```

Comment:
Could you provide an explanation on what the difference is between real, system and user time is? And in what context each of these may be useful?